WordPress/src/main/java/org/wordpress/android/ui/posts/services/PostUploadNotifier.java

```diff
@@ -137,18 +137,19 @@ public void updateNotificationSuccess(PostModel post, SiteModel site, boolean is
         notificationBuilder.setStyle(new NotificationCompat.BigTextStyle().bigText(message));
         notificationBuilder.setAutoCancel(true);
 
+        long notificationId = getNotificationIdForPost(post);
         // Tap notification intent (open the post list)
         Intent notificationIntent = new Intent(mContext, PostsListActivity.class);
         notificationIntent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
         notificationIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
         notificationIntent.putExtra(WordPress.SITE, site);
         notificationIntent.putExtra(PostsListActivity.EXTRA_VIEW_PAGES, post.isPage());
-        PendingIntent pendingIntentPost = PendingIntent.getActivity(mContext, 0,
-                notificationIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+        PendingIntent pendingIntentPost = PendingIntent.getActivity(mContext,
+                (int)notificationId,
```

Comment:
What are the implications of coercing this `long` into an `int` if the `long` is greater than `Integer.MAX_VALUE`? I know the coercion will work, but wasn't sure if does this could break something else.